autospec/config.py

```diff
@@ -293,7 +293,6 @@ def __init__(self, download_path):
             (r"Target '[a-zA-Z0-9\-]' can't be generated as '(.*)' could not be found", 0, None),
             (r"Unable to `import (.*)`", 0, None),
             (r"Unable to find '(.*)'", 0, None),
-            (r"Unknown packages? ['‘]([a-zA-Z0-9\-]*)['’].* in Rd xrefs", 0, 'R'),
```

Comment:
smartquotes ftw

Reply:
Maybe a + instead of * here though, just to be slightly more careful.

Reply:
This is a line I removed... But yes, I agree, if we were keeping the pattern, a `+` would be better.

Reply:
ahhh, quarantine brain. sorry.